src/ngraph/function.cpp

```diff
@@ -38,6 +48,7 @@ Function::Function(const ResultVector& results,
     , m_instance_id(m_next_instance_id.fetch_add(1))
     , m_name(name)
     , m_unique_name("Function_" + to_string(m_instance_id))
+    , m_topological_sorter(topological_sort_wrapper)
```

Comment:
Does `m_topological_sorter(topological_sort<std::vector<std::shared_ptr<Node>>>)` work?

Reply:
Yes it does work, no wrapper needed. Changed